test/init/generator/init-inquirer.test.js

```diff
@@ -42,5 +44,10 @@ describe('init', () => {
             expect(pkgJson['scripts']['build'] == 'webpack').toBeTruthy();
         };
         expect(pkgJsonTests).not.toThrow();
+
+        // Check we are not logging twice
+        if (process.platform !== 'win32') {
```

Comment:
We should stop to adding test with skipping on widows, what is the problem here?

Reply:
The respective assertion fails on windows; need to investigate why that happens.

Reply:
Please look at this, it is real bad when we skip tests on windows